main_ui.py

```diff
@@ -0,0 +1,59 @@
+"""
+    Streamlit Simple Calculator.
+
+    Author: Putri Khalilah Kamaluddin
+    Date: 2023/12/05
+    Description: This file is used to launch a minimal streamlit web 
+	application. A simple calculator that calculates 2 numerical values with addition,
+    substraction, multiplication and division operator.
+
+	For further help with the Streamlit framework, see:
+
+	https://docs.streamlit.io/en/latest/
+"""
+
+#Streamlit depends 
+import streamlit as st #  pip install streamlit
+from calculator import addition, multiplication,division,substraction
+
+def main():

```

Comment:
- Application works fine!
- Some saja saja improvement:
1. Use dropdown to choose the operation instead of using integer for operation. 
2. Use button to execute the operation execution.